core/src/io/anuke/mindustry/editor/MapRenderer.java

```diff
@@ -163,4 +199,66 @@ public void dispose(){
             }
         }
     }
+
+    private void drawMesh(IndexedRenderer mesh, Tile tile, TextureRegion textureRegion,
+        int index, Rectangle2D.Float meshRegion, boolean needsRotate) {
+        if(needsRotate){
+            final float rotationDegree = tile.rotation() * 90 - 90;
+            mesh.draw(index, textureRegion,
+                      meshRegion.x,
+                      meshRegion.y,
+                      meshRegion.width,
+                      meshRegion.height,
+                      rotationDegree);
+        }else{
+            mesh.draw(index, textureRegion,
+                      meshRegion.x,
+                      meshRegion.y,
+                      meshRegion.width,
+                      meshRegion.height);
+        }
+    }
+
+    private int getRandomValue(int index, int length) {
```

Comment:
To re-iterate my point from before: None of these methods should exist. `getRegionInstance` is simply more characters than just writing `Core.atlas.find`, while the rest of these methods (and classes) are just clutter. Tiny helper methods like these only harm readability and waste space.